### ISSUE TITLE
Border is getting disappered for "Reset all preferences" button upon getting the focus in High contrast black mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#343]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode to black before launching the application.

1. Launch Xamarin Workbooks application. Which invokes "Create New Workbook" window.
2. Navigate to Tools -> Preferences. Invoke preferences tab. It will open the general tab of preferences.
3. Navigate to "Reset all preferences" button and observe the focus.

### Expected Behavior
Border should not disappear upon setting the focus.
### Actual Behavior
Border is getting disappered for "Reset all preferences" button upon getting the focus.

Note: In High contrast white mode border will remain on "Reset all preferences" button
### User Impact
User will be confused with loss of border
### Screenshots
![7 hc black reset all preferences button](https://user-images.githubusercontent.com/22956130/40722883-253ad14a-643b-11e8-94b0-d8ba86ec9228.PNG)
![7 hc white with border](https://user-images.githubusercontent.com/22956130/40722885-256ac954-643b-11e8-8855-48b85932b437.PNG)
![7 normal mode](https://user-images.githubusercontent.com/22956130/40722886-2595960c-643b-11e8-84ba-d08eb1260240.PNG)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
